nservicebus/testing/index.md

```diff
@@ -66,4 +66,6 @@ snippet:TestingAdditionalDependencies
 
 ## Constructor injected bus
 
+NOTE: The `IBus` interface was deprecated in Version 6, and replaced with the contextual `IMessageHandlerContext` parameter on the `IHandleMessages<T>.Handle()` methods.
```

Comment:
Is there a documentation link we can point to? 
We talk about this in the upgrade guide. But a doco link would be best.